src/Angor/Client/Pages/Spend.razor

```diff
@@ -653,4 +688,31 @@
         await _clipboardService.WriteTextAsync(trxData);
         StateHasChanged();
     }
+    
+    public struct UtxoKey
```

Comment:
There is already such a type its called `Outpoint` use that instead

Reply:
Will check it out 👍